### PR TITLE
[TAS-43] fixed validatiom issues in students and all users

### PR DESCRIPTION
**Pull request by rishigupta1109**
fixed validation issue in user creation

### CODE REVIEW

File: src/pages/Users/Admins/AddNewAdmin.tsx [hunk 1]
```diff
@@ -163,13 +163,20 @@ const AddNewAdmin: React.FC<IAddNewAdmin> = ({ setOpen, open }) => {
...
-      result.roles = result.roles.map((role: any) => {
-        return {
-          id: role.id,
-          from: role.from,
-          to: role.to,
-        };
-      });
+      if (
+        !result?.roles ||
+        !result?.roles?.find((role: any) => role.id === "ROLE_ADMIN")
```
Comment (Anuragtech02) [inline src/pages/Users/Admins/AddNewAdmin.tsx:168]: Extract this into a separate variable like `isRoleAdmin` or something similar 
And for the string `ROLE_ADMIN`, use from constants.ts

File: src/pages/Users/Operators/AddNewOperator.tsx [hunk 2]
```diff
@@ -162,13 +162,20 @@ const AddNewOperator: React.FC<IAddNewOperator> = ({ setOpen, open }) => {
...
-      result.roles = result.roles.map((role: any) => {
-        return {
-          id: role.id,
-          from: role.from,
-          to: role.to,
-        };
-      });
+      if (
+        !result?.roles ||
+        !result?.roles?.find((role: any) => role.id === "ROLE_OPERATOR")
```
Comment (Anuragtech02) [inline src/pages/Users/Operators/AddNewOperator.tsx:167]: Same here

File: src/pages/Users/Teachers/AddNewTeacher.tsx [hunk 3]
```diff
@@ -182,13 +182,20 @@ const AddNewTeacher: React.FC<IAddNewTeacher> = ({ setOpen, open }) => {
...
-      result.roles = result.roles.map((role: any) => {
-        return {
-          id: role.id,
-          from: role.from,
-          to: role.to,
-        };
-      });
+      if (
+        !result?.roles ||
+        !result?.roles?.find((role: any) => role.id === "ROLE_TEACHER")
```
Comment (Anuragtech02) [inline src/pages/Users/Teachers/AddNewTeacher.tsx:187]: Here too

**Review by Anuragtech02 — CHANGES_REQUESTED** [review]
Look for these

**Comment by rishigupta1109** [comment]
ok

**Pull request merged**